tests/test_writing_methods.py

```diff
@@ -15,6 +15,7 @@
 
 CREDENTIALS = ["figo", "figo"]
 BANK_CODE = "90090042"
+CLIENT_ERROR = 1000
```

Comment:
Same as above...

Reply:
Above I would use the stdlib (`httplib`) provided constant, these here should move into a constant-module, but maybe not in this PR.